src/main/java/edu/ucsb/cs156/happiercows/controllers/CommonsController.java

```diff
@@ -46,7 +46,7 @@
   ObjectMapper mapper;
 
   @ApiOperation(value = "Get a list of all commons")
-  @PreAuthorize("hasRole('ROLE_USER')")
+  // @PreAuthorize("hasRole('ROLE_USER')")                   //gets rid of toast spam when not logged in.
```

Comment:
This does get rid of the toast spam, but possibly at the expense of leaking private FERPA protected information i.e. the name and emails of students in the class, at an unprotected endpoint.

If that's the case (I'm checking that now on QA) that's not ok.

But, if it is ok... then have the courage of your convictions.  Don't just comment out the line.  Delete it.

Commenting it out is fine for testing... but once you are committing code "for real", we don't commit commented out code.

Reply:
Ok, I checked.. this is fine ...it doesn't leak information.  So it's good---but instead of commenting out, delete the line for real.

Then I think it's mergeable.

Reply:
Thank you for the warning and the advice.